app/locales/en/translation.json

```diff
@@ -0,0 +1,800 @@
+{
+ "Optimizer": "Optimizer",
+ "Ocr Debugger": "Ocr Debugger",
+ "Subprocess Debugger": "Subprocess Debugger",
+ "Clear language cache": "Clear language cache",
+ "Gear": "Gear",
+ "Heroes": "Heroes",
+ "Hero": "Hero",
+ "Start": "Start",
```

Comment:
Is the EN translation needed? Seems a bit redundant

Reply:
its not needed，but its an reference for other translators. they can copy it and do the translation